groups/postgres/groups.go

```diff
@@ -1230,12 +1235,20 @@ func toGroup(g dbGroup) (groups.Group, error) {
 		}
 	}
 
+	var desc nullable.Value[string]
+	var err error
+	if g.Description.Valid {
+		if desc, err = nullable.ParseString(g.Description.String); err != nil {
+			return groups.Group{}, errors.Wrap(errors.ErrMalformedEntity, err)
+		}
+	}
```

Comment:
We can have code like this 
```suggestion
	var desc string
	if g.Description.Valid {
		desc = g.Description.String
	}
```